### PR TITLE
YOPN-927 VY: Rebuild user and node logging completely

### PR DESCRIPTION
**Pull request by jediandrey**

### CODE REVIEW

File: modules/openy_gc_auth/openy_gc_auth.info.yml [hunk 1]
```diff
@@ -5,3 +5,5 @@ package: Open Y Virtual YMCA
 version: 0.1
 core: 8.x
 core_version_requirement: ^8 || ^9
+dependencies:
```
Comment (anpolimus) [inline modules/openy_gc_auth/openy_gc_auth.info.yml:8]: It should work only if user enabled this module.
We have to keep everything working if openy_gc_log was disabled.

**Review by anpolimus — CHANGES_REQUESTED** [review]

**Comment by hamrant** [comment]
retest

**Review by hamrant — APPROVED** [review]
**TODO** for @hamrant: add the same log for CSV auth

**Comment by hamrant** [comment]
Tested, with disabled module works fine, with enabled no any issues or errors:

Screenshot from 2020-09-28 12-59-27

No regressions logs work as expected.

**Pull request merged**